lib/schemas/mental_status.js

```diff
@@ -0,0 +1,114 @@
+module.exports = {
+  "id": "mental_status",
+  "type": "array",
+  "properties": {
+    "text": {
+      "type": "string"
+    },
+    "identifiers": {
+      "type": "string",
+      "items": {
+        "$ref": "cda_id"
+      }
+    },
+
+    "observation": {
+      "type": "array",
+      "properties": {
+        "date_time": {
+          "$ref": "cda_date"
+        },
+        "identifiers": {
+          "type": "array",
+          "items": {
+            "$ref": "cda_id"
+          }
+        },
+        "value": {
+          "type": "string"
+        },
+      },
+    },
+    "organizer": {
+      "type": "array",
+      "properties": {
+        "date_time": {
+          "$ref": "cda_date"
+        },
+        "identifiers": {
+          "type": "array",
+          "items": {
+            "$ref": "cda_id"
+          }
+        },
+        "value": {
+          "type": "string"
+        },
+      },
+    },
+    "assessment": {
+      "type": "array",
+      "properties": {
+        "date_time": {
+          "$ref": "cda_date"
+        },
+        "identifiers": {
+          "type": "array",
+          "items": {
+            "$ref": "cda_id"
+          }
+        },
+        "value": {
+          "type": "string"
+        },
+      },
+
+      "additionalProperties": false,
+      "required": [
+        // "date_time", "identifiers"
+      ]
+    },
+/*     "organizer": {
+      "type": "object",
+      "properties": {
+        "date_time": {
+          "$ref": "cda_date"
+        },
+        "identifiers": {
+          "type": "string",
+          "items": {
+            "$ref": "cda_id"
+          }
+        },
+      },
+      "additionalProperties": false,
+      "required": [
+        "date_time", "identifiers"
+      ]
+    }, */
+/*     "organizer": {
+      "type": "string",
+      "properties": {
+        "value": {
+          "type": "string"
+        },
+        "date_time": {
+          "$ref": "cda_date"
+        },
+        "identifiers": {
+          "type": "string",
+          "items": {
+            "$ref": "cda_id"
+          }
+        },
+      }, */
+      "additionalProperties": false,
+      "required": [
+        // "value", "date_time", "identifiers"
+      ]
+    },
+  "additionalProperties": false,
+  "required": [
```

Comment:
We found out that empty "required" arrays cause schema compilation failures. So this should be removed.